flake.nix

```diff
@@ -1,43 +1,31 @@
 {
   description = "Home Manager for Nix";
 
-  outputs = { self, nixpkgs }:
-    let
-      # List of systems supported by home-manager binary
-      supportedSystems = with nixpkgs.lib.platforms; linux ++ darwin;
+  inputs.nixpkgs.url = "nixpkgs/nixos-unstable";
```

Comment:
this seems to work but I would rather see "github:nixpkgs/nixos-unstable" that is less magical; same for flake-utils